### PR TITLE
biomass reclaimer suicide

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR 
funny goes CHUNKCHUNKCHKCHKCHUNKBRRRRR
**Screenshots**

**Changelog**

:cl:
- add: You can now suicide into biomass reclaimers.

### CODE REVIEW

**Comment by mirrorcult** [comment]
based

**Comment by Kromabae** [comment]
No video D:

**Comment by EmoGarbage404** [comment]
> No video D:

You kill yourself and then the machine go brchunkchunkchucnknkrbrrrr

File: Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs [hunk 1]
```diff
@@ -93,10 +96,27 @@ public override void Initialize()
...
             SubscribeLocalEvent<ReclaimSuccessfulEvent>(OnReclaimSuccessful);
             SubscribeLocalEvent<ReclaimCancelledEvent>(OnReclaimCancelled);
         }
 
+        private void OnSuicide(EntityUid uid, BiomassReclaimerComponent component, SuicideEvent args)
+        {
+            if (args.Handled)
+                return;
+            args.SetHandled(SuicideKind.Blunt);
+
```
Comment (mirrorcult) [inline Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs:109]: should really only set handled after it actually processes

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Pull request merged**